### PR TITLE
feat(IDX): container image mirror

### PR DESCRIPTION
**Pull request by marko-k0**
Mirror container images from DockerHub to GHCR to avoid DockerHub's rate limit and other issues. It suffices to mirror container image on `pull_request` event when `container-mirror-images.json` is updated. Once the change is merged to master, follow up with adding it to `MODULES.bazel`. We retain full image names `<owner>/image` to give credit to owners.

### CODE REVIEW

File: .github/workflows/container-mirror.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+      - name: Setup Crane
+        run: |
+          curl -fsL https://github.com/google/go-containerregistry/releases/download/v0.20.2/go-containerregistry_Linux_x86_64.tar.gz | tar xz crane
+          chmod +x crane
+          sudo mv crane /usr/local/bin/crane
+
+      - name: Mirror Container
+        run: |
+          # TODO: remove before merge
```
Comment (marko-k0) [inline .github/workflows/container-mirror.yml:67]: Do not forget.

**Review by basvandijk — APPROVED** [review]
I know @nmattia had comments on the approach but I personally find this code not hard to follow so LGTM.
